RELEASE/scripts/autoscend.ash

```diff
@@ -2117,10 +2117,16 @@ boolean LX_craftAcquireItems()
 		}
 	}
 
-	if((my_meat() > 7500) && (item_amount($item[Seal Tooth]) == 0))
+	if(item_amount($item[Seal Tooth]) == 0)
 	{
-		acquireHermitItem($item[Seal Tooth]);
+		//saucerors want to use sealtooth to delay so that mortar shell delivers final blow for weaksauce MP explosion
+		//TODO: add delaying for mortar for other classes in combat and then remove the sauceror requirement here.
```

Comment:
this is a TODO since we need to add costs calculations first. which I am working on in combat2 branch